rir/src/compiler/pir/instruction.h

```diff
@@ -498,18 +499,31 @@ class FLI(LdConst, 0, Effect::None, EnvAccess::None) {
     void printArgs(std::ostream& out, bool tty) override;
 };
 
-class FLIE(LdFun, 1, Effect::Any, EnvAccess::Write) {
+class FLIE(LdFun, 2, Effect::Any, EnvAccess::Write) {
```

Comment:
I guess you mean member (ie. one variable per object and not per class).

any `pir::Value*` *must* be in the arguments. The whole compiler relies on the fact that we can iterate over all arguments. Otherwise e.g. the cleanup pass will not see the use and just delete the value.